dist/licensing/generate-license-and-notice.sh

```diff
@@ -146,6 +148,15 @@ if [ ! -z "$LIBRARIES" ] ; then
 fi
 
 process_dependencies() {
+  echo running mvn license-audit-maven-plugin:notices
+  echo mvn -X org.heneveld.maven:license-audit-maven-plugin:notices \
+        -DlicensesPreferred=Apache-2.0,Apache,EPL-1.0,BSD-2-Clause,BSD-3-Clause,CDDL-1.1,CDDL-1.0,CDDL \
+        -DoverridesFile=$TEMP_METADATA_FILE \
+        -DoutputYaml=true \
+        -DoutputFile=$TEMP_NOTICE_DATA_FILE \
+        $@
```

Comment:
Is this meant to be here, or is it a left over from diagnostic?

Reply:
i left it as it's useful ... happy to remove (ideally would be `set +e` or `+x` ? i can never remember the command echo ... but this is only run a few times a year to update the license files so not a big deal either way imo